services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> Movie:
+    if genres_ids is None and actors_ids is None:
+        return Movie.objects.all()
+    if actors_ids is not None and genres_ids is not None:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids is not None and actors_ids is None:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+    if actors_ids is not None and genres_ids is None:
+        return Movie.objects.filter(actors__id__in=actors_ids)
```

Comment:
it is so hard. Just create some variable queryset and then rewrite this variable and apply filter:
queryset = queryset.filter(.....)
In your function must be only two conditions:
1. When genres_ids is not None
2. When actors_ids is not None